tools/StaticAnalysis/Exceptions/Az.Sql/BreakingChangeIssues.csv

```diff
@@ -47,4 +47,5 @@
 "Microsoft.Azure.PowerShell.Cmdlets.Sql.dll","Microsoft.Azure.Commands.Sql.ThreatDetection.Cmdlet.SetAzureSqlServerThreatDetection","Set-AzSqlServerThreatDetectionPolicy","0","3020","The element type for property 'ExcludedDetectionTypes' has been changed from 'Microsoft.Azure.Commands.Sql.ThreatDetection.Model.DetectionType' to 'System.String'.","Change the element type for property 'ExcludedDetectionTypes' back to 'Microsoft.Azure.Commands.Sql.ThreatDetection.Model.DetectionType'."
 "Microsoft.Azure.PowerShell.Cmdlets.Sql.dll","Microsoft.Azure.Commands.Sql.ThreatDetection.Cmdlet.SetAzureSqlServerThreatDetection","Set-AzSqlServerThreatDetectionPolicy","0","2110","The element type for parameter 'ExcludedDetectionType' has been changed from 'Microsoft.Azure.Commands.Sql.ThreatDetection.Model.DetectionType' to 'System.String'.","Change the element type for parameter 'ExcludedDetectionType' back to 'Microsoft.Azure.Commands.Sql.ThreatDetection.Model.DetectionType'."
 "Microsoft.Azure.PowerShell.Cmdlets.Sql.dll","Microsoft.Azure.Commands.Sql.ManagedInstance.Cmdlet.GetAzureSqlManagedInstance","Get-AzSqlInstance","0","1050","The parameter set '__AllParameterSets' for cmdlet 'Get-AzSqlInstance' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'Get-AzSqlInstance'."
-"Microsoft.Azure.PowerShell.Cmdlets.Sql.dll","Microsoft.Azure.Commands.Sql.ManagedInstance.Cmdlet.NewAzureSqlManagedInstance","New-AzSqlInstance","0","1050","The parameter set '__AllParameterSets' for cmdlet 'New-AzSqlInstance' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'New-AzSqlInstance'."
\ No newline at end of file
+"Microsoft.Azure.PowerShell.Cmdlets.Sql.dll","Microsoft.Azure.Commands.Sql.ManagedInstance.Cmdlet.NewAzureSqlManagedInstance","New-AzSqlInstance","0","1050","The parameter set '__AllParameterSets' for cmdlet 'New-AzSqlInstance' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'New-AzSqlInstance'."
+"Microsoft.Azure.PowerShell.Cmdlets.Sql.dll","Microsoft.Azure.Commands.Sql.ManagedDatabase.Cmdlet.RestoreAzureRmSqlManagedDatabase","Restore-AzSqlInstanceDatabase","0","2020","The cmdlet 'Restore-AzSqlInstanceDatabase' no longer supports the type 'Microsoft.Azure.Commands.Sql.ManagedDatabase.Model.AzureSqlManagedDatabaseModel' for parameter 'InputObject'.","Change the type for parameter 'InputObject' back to 'Microsoft.Azure.Commands.Sql.ManagedDatabase.Model.AzureSqlManagedDatabaseModel'."
```

Comment:
This is OK here because the new interface of `-InputObject` is the parent of the old one

Reply:
Yes, that is what I thought.